packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -27,11 +27,11 @@ Source1:        ecs.conf
 
 BuildRequires:  golang >= 1.7
 
-Requires:       docker >= 1.6.0, docker <= 17.03.2ce
+Requires:       docker = 17.03.2ce
 Requires:       upstart
 Requires:       iptables
 Requires:       procps
-Requires(post): docker >= 1.6.0
+Requires:       dhclient
```

Comment:
We'll want to update both our SUSE and Ubuntu packages as well to have the new dependency.

@rjschwei I assume that SUSE has dhclient in a package; can you confirm that this will be a requirement that can be satisfied on SUSE?

Reply:
No, for SLES the network handling is done with wicked, so the dhcp requirement can be met with a different implementation, however the apparent dependence on the format and location of the lease information file is going to be a problem.

Reply:
Is dhclient available at all in SLES? If so, it'd be easiest to let it come in as a dependency here. The corresponding [amazon-ecs-agent work](https://github.com/aws/amazon-ecs-agent/tree/enis) is fairly tightly coupled to dhclient right now.

Using dhclient for the ENI support wouldn't require the use of dhclient for system-wide network configuration, assuming the two packages can be installed side-by-side. There may be additional work required to ensure they don't step on each other when task ENIs are attached.

Reply:
No, dhclient is not available for SLES, not packaged and supported by SUSE anyway.